### DISCUSSION TITLE
could not find function "launchSPATAData"

### DISCUSSION THREAD

**Discussion by Sam-Brain** [theMILOlab/SPATA2#148] [Q&A]
could not find function "launchSPATAData"

**Comment by Sam-Brain** [comment]
I tried to use the shiny app of SPATAData, but it is not loading. I followed https://themilolab.github.io/SPATA2/articles/spata-v2-installation.html and updated the R and R studio and all packages, still have the problem of starting the shiny app via ''launchSPATAData''.
All needed packages (SPATA2, Seurat and confuns) are installed. Can you help? Does anybody have the same problem?

**Comment by kueckelj** [comment]
Hi, SPATAData has undergone some changes in preparation for SPATA2v3 which we are about to release in the next days. You will find information about how to download samples of interest on the updated website once we have published the new version.

**Reply by Sam-Brain** [reply]
Thank you so much and congratulations on the paper 👍

**Reply by kueckelj** [reply]
Thank you. It's published and you should be able to download everything now.
